handbook/repository/aws-secrets.md

```diff
@@ -0,0 +1,135 @@
+# AWS Secrets
+
+## Prerequisites
+
+You will need `awscli` installed
+
+```bash
+brew install awscli
```

Comment:
rather point to this url in order to avoid being os agnostic:
https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html